src/helpers/dateUtil.js

```diff
@@ -23,9 +23,19 @@ function diff (date1, date2, period) {
     .diff(date1, period)
 }
 
+function nowIsBetween (date1, date2) {
+  return moment().isBetween(date1, date2)
+}
+
+function newDate (date, format) {
+  return moment(date, format)
```

Comment:
This doesn't return a date, right?

It returns a moment date. I wouldn't expose that out of the dateUtil